srcpkgs/lyx/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'lyx'
 pkgname=lyx
-version=2.3.4.4
+version=2.3.5.1
 revision=1
+wrksrc="${pkgname}-2.3.5-1"
```

Comment:
I think you can do a bash substitution with the version here. I don't know the exact syntax right now, though.

Reply:
it is `${version%.*}-${version##*.}`

Reply:
Done, thanx @ericonr and @Chocimier !